src/sidebar/editor.js

```diff
@@ -17,22 +17,49 @@ function customizeEditor(editor) {
   });
   
   document.addEventListener('dragover', () => {
-    const mainEditor = document.querySelector('.ck-editor__main');
-    mainEditor.style.border = '1px solid green';
-    mainEditor.style.backgroundColor = '#dcd6da';
+    const theme = browser.storage.local.get(['theme']);
+    theme.then(function getTheme(data) {
+      if (data.theme === 'default') {
```

Comment:
I would recommend doing that with a class that we can define in both styles.css and styles-dark.css rather than having this kind of if around the theme here.

Reply:
@Natim , do you mean that instead of setting colors here , I have to just add a class and then add the class and remove the class upon requirement right ?

Reply:
Yes

Reply:
Yo got it

Reply:
Have a look now